charts/bitwardenrs/templates/deployment.yaml

```diff
@@ -42,6 +42,9 @@ spec:
           envFrom:
             - configMapRef:
                 name: {{ $fullName }}
+            {{- with .Values.envFrom }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
```

Comment:
Instead of envFrom, which would require an all-inclusive addition that already exists, thoughts on the same format but just env? so that users can add individual additional k/v?

Reply:
The all-inclusive addition in this chart only supports configmap. As the comment in values.yaml suggests, I was looking for a way to read DATABASE_URL from a secret as it contains database password.

My first idea was
```yaml
envFromSecrets:
  DATABASE_URL:
    secret: bitwardenrs-db
    key: databaseUrl
```
but then I considered that the direction of this repository is to use the `common` template for charts or at least a common structure for values and decided to take the same approach.

And of course all of that is just an excuse to avoid adding proper database settings object :slightly_smiling_face:

Reply:
I added an external database config object so my configuration no longer requires `envFrom`.
It's still the only way to configure Duo 2FA or HIBP API key, so I'd prefer to have a support for reading arbitrary environment variables from secret. If you have any thoughts on the format for configuration, I can try implementing that.